### ISSUE TITLE
Remove remnants of Travis

### ISSUE THREAD

**Issue by zsunberg** [JuliaPOMDP/POMDPs.jl#363]
We need to go through and remove all .travis.yml files and travis badges. This would be easy to do - just need a volunteer to do it.

**Comment by dylan-asmar** [comment]
@zsunberg is there a particular workflow preference you have when we update this? In [SARSOP.jl](https://github.com/JuliaPOMDP/SARSOP.jl), I added separate workflows for documentation and code coverage of tests. I don't mind making similar changes as I come across different repos, but wanted to make sure that separate GitHub actions are ok or if you prefer one large workflow.

**Comment by zsunberg** [comment]
@dylan-asmar awesome! thanks for your work in SARSOP!

I tend to try to follow what PkgTemplates does, just because that will be the most common thing for the community. Looks like they have it in one file: https://github.com/invenia/PkgTemplates.jl/blob/master/templates/github/workflows/CI.yml

**Comment by zsunberg** [comment]
If you wanted to get really fancy and really embody the CS virtue of lazyness, you could write a script to do this for all the packages (possibly using the gh command line tool, but I don't think that is strictly needed).

**Comment by dylan-asmar** [comment]
Making some progress:
 - https://github.com/JuliaPOMDP/SARSOP.jl/pull/60
 - https://github.com/JuliaPOMDP/LaserTag.jl/pull/15
 - https://github.com/JuliaPOMDP/POMCPOW.jl/pull/35
 - https://github.com/JuliaPOMDP/POMDPModels.jl/pull/87
 - https://github.com/JuliaPOMDP/TabularTDLearning.jl/pull/26
 - https://github.com/JuliaPOMDP/BasicPOMCP.jl/pull/32

**Comment by zsunberg** [comment]
Awesome, @dylan-asmar , this is really great!

**Comment by mykelk** [comment]
Thanks so much @dylan-asmar ! I think you can merge these when ready.

**Comment by zsunberg** [comment]
Yeah, at this point, @dylan-asmar , I think you can just go ahead and merge all pull requests that you have yourself. Just make sure tests pass, etc. The only reason not to would be if you have specific questions that you're not sure how to answer.

Do you have the necessary github permissions to do that?

**Comment by dylan-asmar** [comment]
I believe I have the permissions. As I'm going through the repos I'm also making minor changes from time to time. I'll ping you @zsunberg on those to just make sure it doesn't mess something up.

**Comment by dylan-asmar** [comment]
For https://github.com/JuliaPOMDP/POMDPGallery.jl and https://github.com/JuliaPOMDP/POMDPExamples.jl, what are the thoughts on updating these with the https://github.com/JuliaPOMDP/POMDPs.jl/issues/241 and https://github.com/JuliaPOMDP/POMDPs.jl/issues/417 still open?

**Comment by zsunberg** [comment]
I don't think we should maintain POMDPExamples. The important tutorials there should be moved to the main POMDPs.jl documentation (i.e. I still think we should fix #417).

I'm less sure about POMDPGallery. I guess since we list the problem repos in the POMDPs.jl README now, the best thing would be to move the examples from POMDPGallery to the individual project repos, but I am not 100% sure.

But that is a lot of work that is probably not the highest priority. If someone can do it, great! but it is not a disaster in the state it is in now.

**Comment by mykelk** [comment]
I think we should keep POMDPGallery. I use it quite a bit when demoing POMDPs.jl. I'm okay with POMDPExamples eventually being merged into the documentation of POMDPs.jl.

**Comment by dylan-asmar** [comment]
I think POMDPGallery is a great tool to show off POMDPs.jl. However, I think existing as a separate package doesn't add a lot of value. I think it would be better to have it as part of the docs of POMDPs.jl. 

One option would be to have the scripts as examples and then the gifs get created at documentation generation. Another option is to just embed the gifs into the docs and make sure we update them at each new release of POMDPs.jl.

An alternative thought is to create a GitHub action with the problem repos where when updates to the problems occur, it autogenerates the gifs and submits a PR to this POMDPGallery repo.

**Comment by mykelk** [comment]
I'm good with moving them into the docs. I'd really like to make it easy for people to find a page with all of these examples. If in the docs, then I think we should link directly to that location from the readme.md.

**Comment by dylan-asmar** [comment]
If we agree to not update POMDPExamples and POMPGallery (move both to docs), then I think this issue is almost ready to be closed out. 

The only repo that I think is left is POMDPLinter.jl. When updating that one, I ran into an issue where tests were failing (without updates). I opened an issue on that repo and once we address that issue we can merge the travis_cleanup_ci_updates branch. 

@zsunberg , can you take a look at the issue on POMDPLinter.jl and provide any feedback/guidance? I don't mind making changes, but I wasn't sure if I was missing something there.

Is anybody aware of another repo that might have slipped by me and is still using Tavis-CI vs GitHub actions?

**Comment by dylan-asmar** [comment]
I think we are finally complete with this one. 
If there are other repos that need to be addressed, please re-open this issue.
